wondrous-app/components/OnboardingDao/CreateDao/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from './component';
```

Comment:
same with AddImages, if there's no other exports from the main we can simply put the component itself in the index file